monkey/monkey_island/cc/models/monkey.py

```diff
@@ -43,22 +43,32 @@ class Monkey(Document):
     tunnel = ReferenceField("self")
     command_control_channel = EmbeddedDocumentField(CommandControlChannel)
     aws_instance_id = StringField(required=False)  # This field only exists when the monkey is running on an AWS
+
     # instance. See https://github.com/guardicore/monkey/issues/426.
 
+    @staticmethod
+    def __ring_key__():
```

Comment:
I think this is called by the bug of previous version. If staticmethod is the only usage of ring of this class, this method will not be required.
If this object is used as a cache key, then this implementation can cause bug.

Next code will be correct.
```
def __ring_key__(self):
    return sef.guid
````

Reply:
This class is not used as a parameter to other cached functions. Can we delete this, then? 

Also thanks a million for all your help :)

Reply:
Yes, then deleting will be better not to make a future confusion. I created #484

Reply:
I've merged it. Thanks!!!